src/theme/globals.css

```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
```

Comment:
May be a better idea to save these locally and reference them so that we're not stuck making a fetch to get fonts to load.

Reply:
makes sense and agreed